packages/phrases/src/locales/de/translation/admin-console/connector-details.ts

```diff
@@ -27,6 +27,8 @@ const connector_details = {
     'This connector is in-use in your sign in experience. By deleting, <name/> sign in experience will be deleted in sign in experience settings.', // UNTRANSLATED
   in_used_passwordless_deletion_description:
     'This {{name}} is in-use in your sign-in experience. By deleting, your sign-in experience will not work properly until you resolve the conflict.', // UNTRANSLATED
```

Comment:
should we also mention "and you will need to reconfigure it.." for these two prompts?

Reply:
Updated.